config/routes.rb

```diff
@@ -42,5 +42,6 @@
       put "toggle_admin", on: :member
     end
   end
+  match "(errors)/:status", to: "errors#show", constraints: { status: /\d{3}/ }, via: :all
```

Comment:
Does that include also the errors raised in other circumstances (like the 404 error trying to view a non existing team)?